index.bs

```diff
@@ -1981,7 +1991,6 @@ following table:
  The <dfn id="rs-byob-request-respond-with-new-view" method
  for="ReadableStreamBYOBRequest">respondWithNewView(|view|)</dfn> method steps are:
 
- 1. If |view|.\[[ByteLength]] is 0, throw a {{TypeError}} exception.
  1. If |view|.\[[ViewedArrayBuffer]].\[[ArrayBufferByteLength]] is 0, throw a {{TypeError}}
     exception.
```

Comment:
Hmm, I suppose this check is now made redundant by this check in `ReadableByteStreamControllerRespondWithNewView`:
> 4. If firstDescriptor’s buffer byte length is not view.[[ViewedArrayBuffer]].[[ByteLength]], throw a RangeError exception.

...given that a pull-into's buffer can never be zero length.

I guess we can delete this?

Reply:
Yeah, I guess so, although we'd want to update the tests to change the exception type expected.